### ISSUE TITLE
Replace standard `at-rule-no-unknown` by the one from stylelint-scss

### ISSUE THREAD

**Issue by romulof** [bjankord/stylelint-config-sass-guidelines#18]
Some presets like `stylelint-config-recommended` enable stylelint's standard `at-rule-no-unknown`, that blacklists all SASS control at-rules.

`stylelint-scss` package has a replacement rule:
https://github.com/kristerkari/stylelint-scss/tree/master/src/rules/at-rule-no-unknown

To fix it you need only to add these 2 rules:
```
"at-rule-no-unknown": null,
"scss/at-rule-no-unknown": true,
```

**Comment by bjankord** [comment]
Hi @romulof, I'm open to adding this but want to know some more on why `"at-rule-no-unknown": null,` needs to be set. Currently, stylelint-config-sass-guidelines does not use `at-rule-no-unknown`. 

Since it is not currently used, it seems like we would only need to add 
```
"scss/at-rule-no-unknown": true,
```

I'm not entirely sure if we need to set `at-rule-no-unknown` to `null` as well
I'm curious if maybe @kristerkari can answer why he added the `"at-rule-no-unknown": null,` before the `scss/at-rule-no-unknown`.

**Comment by kristerkari** [comment]
> I'm curious if maybe @kristerkari can answer why he added the "at-rule-no-unknown": null, before the scss/at-rule-no-unknown.

You only need to set `"at-rule-no-unknown": null` if you are extending some config that has it enabled.  You are correct that in this case setting `"scss/at-rule-no-unknown": true` would be sufficient.

**Comment by bjankord** [comment]
Thanks for the quick reply @kristerkari. @romulof I'm cool with adding `"scss/at-rule-no-unknown": true,` I'll plan on getting this into the v5.0.0 release.

**Comment by romulof** [comment]
Stylelint's standard `at-rule-no-unknown` is incompatible with SASS control at-rules, that's why it's best to be manually turned off. If it was the case of an external rule I'd totally agree with leaving it out, but since it's an embedded rule, I think it's less error-prone to disable it.

**Comment by bjankord** [comment]
This has been resolved in 70b2e0f90a22b06170b54b3e3257a8173dd61b4b

**Comment by bjankord** [comment]
This is now available in the v5.0.0 release.
